package.json

```diff
@@ -1,14 +1,13 @@
 {
   "name": "uuid",
-  "version": "3.0.0",
+  "version": "3.1.0",
```

Comment:
no version changes in PRs, those happen after once ready to release (with changelog updates accordingly).